src/chunk.c

```diff
@@ -83,13 +83,15 @@ static Sample *ChunkGetSample(Chunk *chunk, int index) {
 timestamp_t Uncompressed_GetLastTimestamp(Chunk_t *chunk) {
     if (unlikely(((Chunk *)chunk)->num_samples == 0)) { // empty chunks are being removed
         RedisModule_Log(mr_staticCtx, "error", "Trying to get the last timestamp of empty chunk");
+        return 0;
     }
     return ChunkGetSample(chunk, ((Chunk *)chunk)->num_samples - 1)->timestamp;
 }
 
 double Uncompressed_GetLastValue(Chunk_t *chunk) {
     if (unlikely(((Chunk *)chunk)->num_samples == 0)) { // empty chunks are being removed
         RedisModule_Log(mr_staticCtx, "error", "Trying to get the last value of empty chunk");
+        return 0;
```

Comment:
is it possible to add test for this scenario @OfirMos ?

Reply:
Nope, it's a dead code, meant to catch errors